app/packs/src/apps/mydb/elements/details/researchPlans/researchPlanTab/ResearchPlanDetailsFieldTableMeasurementExportModal.js

```diff
@@ -154,7 +154,7 @@ class ResearchPlanDetailsFieldTableMeasurementExportModal extends Component {
           </Modal.Title>
         </Modal.Header>
         <Modal.Body className='measurementExportModal__body'>
-          <table className="table">
+          <Table className="table">
```

Comment:
Is this class "table" necessary?

Reply:
Probably not. Will make necessary changes